webapp/channels/src/components/admin_console/team_channel_settings/convert_and_remove_confirm_modal.tsx

```diff
@@ -62,21 +71,37 @@ const ConvertAndRemoveConfirmModal = ({show, onConfirm, onCancel, displayName, t
     );
 
     const message = (
-        <div>
```

Comment:
did you verify this div was not needed at all? not even being used by the css to apply hierarchy styles?

Reply:
@pvev Yes, I checked, and the div wasn’t needed for styling. It was there because JSX requires a single parent element to wrap multiple tags. A fragment works the same way without affecting anything.